WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -725,6 +776,29 @@ private void submitSearchQuery(@NonNull String query) {
         }
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onReaderSitesSearched(OnReaderSitesSearched event) {
+        if (!isAdded()) {
+            return;
+        }
+
+        mRecyclerView.setRefreshing(false);
+        showLoadingProgress(false);
+
+        ReaderSiteSearchAdapter adapter = getSiteSearchAdapter();
+        if (event.isError()) {
+            adapter.clear();
```

Comment:
Not sure how error handling is handled in this app...should the user see a notification?